.github/workflows/perf.yml

```
@@ -27,6 +40,9 @@ jobs:
     name: Benchmarks (Python 3.11)
     runs-on: ubuntu-latest
     timeout-minutes: 25
+    env:
+      BENCHMARK_ARTIFACT_DIR: ${{ runner.temp }}/modelaudit-benchmarks
```

actionlint (1.7.11)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)